front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -505,26 +529,28 @@ export default function AssistantBuilder({
                       ? "Add instructions or tools to Preview"
                       : "Preview"
                   }
-                  variant="primary"
+                  variant="highlight"
                   disabled={isBuilderStateEmpty}
-                  className={classNames(
-                    isPreviewButtonAnimating ? "animate-breathing-scale" : ""
+                  className={cn(
+                    isPreviewButtonAnimating && "animate-breathing-scale"
                   )}
                 />
               )}
               {rightPanelStatus.tab === null && template !== null && (
-                <div className="flex flex-col gap-3 rounded-full border border-structure-200 p-4">
-                  <IconButton
+                <div className="flex flex-col gap-3">
+                  <Button
```

Comment:
Why not keep the IconButton here and change the props accordingly (same for the other 2 occurrences)?

Reply:
IconButton is deprecated (we're rolling it out).

Reply:
makes sense, I'll take it